packages/e2e-test-utils-playwright/src/editor/publish-post.ts

```diff
@@ -12,7 +12,7 @@ import type { Editor } from './index';
 export async function publishPost( this: Editor ) {
 	await this.page.click( 'role=button[name="Publish"i]' );
 	const publishEditorPanel = this.page.locator(
-		'role=region[name="Publish editor"i]'
+		'role=region[name="Editor publish"i]'
```

Comment:
Yep, I think this is a typo 😅 . It's causing the `isPublishEditorVisible` check always to return false. Thanks for fixing it!